patrickarlt/leaflet-virtual-grid
# trap for when map.getPixelWorldBounds() returns null

resolves: https://github.com/Esri/esri-leaflet/issues/1106

also:
  * have travis use latest stable node perpetually
  * remove .npmignore (in favor of pkg.files)
  * pkg.prepublish > pkg.prepare
  * make sure leftover tape files arent committed
